hdtv/database/PGAALibraries.py

```diff
@@ -16,18 +16,25 @@ class PGAAGamma(Gamma):
     k0_comp is the comparator element for k0 values which k0 should be normed to 1.0
         given as tuple (Z, A) (default: 1-H)
     """
+
     k0_norm = None
-    __slots__ = ("ID", "nuclide", "energy", "sigma",
-                 "intensity", "halflife", "_k0")
+    __slots__ = ("ID", "nuclide", "energy", "sigma", "intensity", "halflife", "_k0")
 
-    def __init__(self, nuclide, energy, sigma=None, intensity=None,
-                 k0=None, halflife=None, k0_comp=(1, 1)):
+    def __init__(
+        self,
+        nuclide,
+        energy,
+        sigma=None,
+        intensity=None,
+        k0=None,
+        halflife=None,
+        k0_comp=(1, 1),
+    ):
 
         super(PGAAGamma, self).__init__(nuclide, energy, sigma, intensity)
         self.halflife = halflife
         self._k0 = k0  # TODO
-        if nuclide == Nuclides(k0_comp[0], k0_comp[1])[
-                0] and not PGAAGamma.k0_norm:
+        if nuclide == Nuclides(k0_comp[0], k0_comp[1])[0] and not PGAAGamma.k0_norm:
```

Comment:
Codacy found an issue: ['Nuclides' may be undefined, or defined from star imports: hdtv.database.common (F405)](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)